tests/test_api.py

```diff
@@ -30,139 +32,80 @@ def bliss_client(cwd, cfg):
 @pytest.fixture(scope="class")
 def cached_data_path_api(bliss_client):
     bliss_client.cached_data_path = bliss_client.cwd + "/data/cached_dataset"
-    bliss_client.generate(n_batches=3, batch_size=8, max_images_per_file=16)
+    bliss_client.generate(n_batches=2, batch_size=5, max_images_per_file=5)
```

Comment:
Why 5? I think we should stick to at least 8. And reason for `max_images_per_file` being 2 x `batch_size` is also as a sanity check that two batches get "flattened"/itemized and saved as expected.

Reply:
5 is the minimum value we can use since batch norm requires `batch_size > 4`. Is there anything 8 would give us that 5 doesn't? Tests are taking longer again with multi band so any time saved is useful. I can change it to 3x5 instead if we want to keep testing the max per file behavior.

Reply:
Yea 5 works, I just thought 8 is nicer since it’s even (and the lowest power-of-2 that is >4).